oic.r.acl2.raml

```diff
@@ -9,6 +9,16 @@ traits:
       queryParameters:
         if:
           enum: ["oic.if.baseline"]
+  - ace-filtered:
+      usage: Apply this to any method that can filter ACEs
+      description: Some requests only operate on a subset of ACEs
+      queryParameters:
```

Comment:
ACE2 doesn't define subjectuuid, as it's attempting to allow the subject/target of the ACE to be either a role, a device/host, or a wildcard. And while creds have subjects and subjectuuids, the subjects of creds and ACEs are fundamentally different. So the lack of consistency isn't a big issue, IMHO.